src/apis/KapacitorApi.ts

```diff
@@ -16,10 +18,10 @@ export default class KapacitorApi {
         return test ? test.level : 'OK';
     }
 
-    public static async taskVariables(taskName: string): Promise<any> {
-        const response = await fetch(`http://82.140.0.78:9092/kapacitor/v1/tasks/${taskName}`);
-        const topic = await response.json();
-        return topic.vars;
+    public static async tasks(): Promise<KapacitorTask[]> {
```

Comment:
The return type should be specified through an interface.

Reply:
What do you mean exactly? `KapacitorTask[]` is the interface of the return type.